dynamo/variables.tf

```diff
@@ -67,3 +67,12 @@ variable "ttl_attribute_name" {
   description = "This attribute name will be used to enable ttl on the table"
   default     = null
 }
+
+variable "stream_enabled" {
+  type    = bool
+  default = false
+}
+
+variable "stream_view_type" {
+  default = "NEW_IMAGE"
```

Comment:
Just curious, how come we're not using `NEW_AND_OLD_IMAGES`?

Reply:
That was my first choice too but the ticket said new image. https://national-archives.atlassian.net/browse/DR2-1711 

Ineither case, the actual value is set from the environments

Reply:
Ah, I see.